scripts/us_bls/jolts/BLSJolts.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pandas as pd
+import urllib.request
+OUTPUT_MCF_TO_CONSOLE = True
+
+### Step 1) Data Download
+# Dataset being processed: https://download.bls.gov/pub/time.series/jt/
+# Data split across several white space separate files that may have incomplete rows
+def web_to_pandas(url, sep):
+  '''
+    Helper to load incomplete whitespace separated data file into pandas
+  '''
+  df = None
+  remote_web_page = urllib.request.urlopen(url)
```

Comment:
I added an assert statement below to get the response code. In the case of a timeout or page not found the code would error and fail, which it would need to do unless the resource should be tried again. Should I be explicitly logging if there is an error?